.github/workflows/main.yml

```diff
@@ -37,9 +37,9 @@ jobs:
           - Pharo32-stable
           - Pharo32-alpha
           - Pharo32-3.0
+          - Moose64-10
           - Moose64-9.0
           - Moose64-8.0
-          - Moose64-7.0
```

Comment:
Let's drop `Moose64-8.0` instead of `Moose64-7.0` so that we still test the oldest supported version.

Reply:
Moose 7 is not supported anymore. Moose 8 is.

Reply:
Ok, good!